### PR TITLE
Doc: add playground examples everywhere

### PR DESCRIPTION
**Pull request by stifoon**

### CODE REVIEW

**Comment by stifoon** [comment]
Preview http://stifoon.github.io/navitia/

File: documentation/slate/source/includes/welcome.md [hunk 1]
```diff
@@ -70,19 +71,19 @@ At some point you will want to read:
 - [transport public lexicon](#lexicon)
 - [Ressources overview](#ressources)
 
-I'm only a human
-----------------
+Navitia for human
+------------------
+>[Try a basic request on Navitia playground](http://canaltp.github.io/navitia-playground/play.html?request=https%3A%2F%2Fapi.navitia.io%2Fv1%2F%2Fcoverage%2Fsandbox%2F%3F&token=3b036afe-0110-4202-b9ed-99718476c2e0)
```
Comment (TeXitoi) [inline documentation/slate/source/includes/welcome.md]: bad link: https://api.navitia.io/v1//coverage/sandbox//?
double / are bad.

File: documentation/slate/source/includes/apis.md [hunk 2]
```diff
@@ -1029,157 +1045,134 @@ Other parameters to come...
...
+This endpoint gives you access to schedules of routes, with a response made
+of an array of [route_schedule](#route-schedule), and another one of [note](#note). You can
+access it via that kind of url: <https://api.navitia.io/v1/{a_path_to_a_resource}/route_schedules>
+
+### Accesses
+
+| url | Result |
+|-------------------------------------------------|---------------------------------------------------------------------------------------------------------------|
+| `/coverage/{resource_path}/route_schedules`  | List of the entire route schedules for a given resource                                                       |
+| `/coverage/{lon;lat}/route_schedules`        | List of the entire route schedules for coordinates                                                       |
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: as we saw in the issue https://github.com/CanalTP/navitia/issues/1651, this url is not working.
right now, should be `/coverage/{lon;lat}/coords/{lon;lat}/route_schedules`

File: documentation/slate/source/includes/apis.md [hunk 3]
```diff
@@ -1029,157 +1045,134 @@ Other parameters to come...
...
+
+This endpoint gives you access to schedules of routes, with a response made
+of an array of [route_schedule](#route-schedule), and another one of [note](#note). You can
+access it via that kind of url: <https://api.navitia.io/v1/{a_path_to_a_resource}/route_schedules>
+
+### Accesses
+
+| url | Result |
+|-------------------------------------------------|---------------------------------------------------------------------------------------------------------------|
+| `/coverage/{resource_path}/route_schedules`  | List of the entire route schedules for a given resource                                                       |
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: missing a coverage name (or {lon;lat})

File: documentation/slate/source/includes/apis.md (hunk 2 above)
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: for those url, there is the same problem for stop_schedules, places nearby, departures, arrivals.
In "Public Transportation Objects exploration" we mention regions, and not around coordinates, which is more correct

File: documentation/slate/source/includes/welcome.md [hunk 4]
```diff
@@ -24,16 +24,15 @@ A fake token is used in examples below: be aware that this token is really limit
 
 First step
 ---------------
-``` shell
-#your token is in your confirmation mail. It sounds like "3b036afe-0110-4202-b9ed-99718476c2e0"
-```
+> Your token is in your confirmation mail.
```
Comment (pbougue) [inline documentation/slate/source/includes/welcome.md]: This is not the case anymore : you have to connect to navitia.io portal to retrieve your token, if I'm not mistaken.
@alcalyn may confirm that, but I'm pretty sure

Comment (alcalyn) [inline documentation/slate/source/includes/welcome.md]: Yep the token is not in the confirmation mail when subscribing to navitia.io

**Comment by stifoon** [comment]
Poke @pbougue @antoine-de @TeXitoi 
:)

**Pull request merged**